Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultBrushToolOptionsView.kt

```diff
@@ -0,0 +1,166 @@
+    private val buttonRect: Chip
+    private val brushToolPreview: BrushToolPreview
+    private var brushChangedListener: OnBrushChangedListener? = null
+
+    companion object {
+        private val TAG = DefaultBrushToolOptionsView::class.java.simpleName
+    }
+
+    init {
+        val inflater = LayoutInflater.from(rootView.context)
+        val brushPickerView = inflater.inflate(R.layout.dialog_pocketpaint_stroke, rootView, true)
+        brushPickerView.apply {
+            buttonCircle = findViewById(R.id.pocketpaint_stroke_ibtn_circle)
+            buttonRect = findViewById(R.id.pocketpaint_stroke_ibtn_rect)
+            brushWidthSeekBar = findViewById(R.id.pocketpaint_stroke_width_seek_bar)
+            brushWidthSeekBar.setOnSeekBarChangeListener(OnBrushChangedWidthSeekBarListener())
+            brushSizeText = findViewById(R.id.pocketpaint_stroke_width_width_text)
+            brushToolPreview = findViewById(R.id.pocketpaint_brush_tool_preview)
+        }
+        brushSizeText.filters = arrayOf<InputFilter>(DefaultNumberRangeFilter(MIN_VAL, MAX_VAL))
+        buttonCircle.setOnClickListener { onCircleButtonClicked() }
+        buttonRect.setOnClickListener { onRectButtonClicked() }
+        brushSizeText.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) =
+                Unit
+
+            override fun onTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) = Unit
+
+            override fun afterTextChanged(editable: Editable) {
+                val sizeText = brushSizeText.text.toString()
+                val sizeTextInt: Int = try {
+                    sizeText.toInt()
+                } catch (exp: NumberFormatException) {
+                    exp.localizedMessage?.let {
+                        Log.d(TAG, it)
+                    }
+                    MIN_BRUSH_SIZE
+                }
+                brushWidthSeekBar.progress = sizeTextInt
+            }
+        })
+    }
+
+    private fun onRectButtonClicked() {
+        updateStrokeCap(Cap.SQUARE)
+        buttonRect.isSelected = true
+        buttonCircle.isSelected = false
+        invalidate()
+    }
+
+    private fun onCircleButtonClicked() {
+        updateStrokeCap(Cap.ROUND)
+        buttonCircle.isSelected = true
+        buttonRect.isSelected = false
+        invalidate()
+    }
+
+    override fun setCurrentPaint(paint: Paint) {
+        if (paint.strokeCap == Cap.ROUND) {
+            buttonCircle.isSelected = true
+            buttonRect.isSelected = false
+        } else {
+            buttonCircle.isSelected = false
+            buttonRect.isSelected = true
+        }
+        brushWidthSeekBar.progress = paint.strokeWidth.toInt()
+        brushSizeText.setText(
+            String.format(
+                Locale.getDefault(), "%d",
+                paint.strokeWidth
+                    .toInt()
+            )
+        )
```

Comment:
```suggestion
        brushSizeText.setText("{paint.strokeWidth.toInt()}")
```
@ThorstenBandel, what do you think about this?

Reply:
I discussed it with Thorsten, this part is basically handling the case of showing the number in devices' default language, which the suggested code won't handle :)

Reply:
It's only a number, but ok 👍

Reply:
Yeah but this is to handle non arabic numerals, like Hindi :)